pycaret/loggers/comet_logger.py

```diff
@@ -0,0 +1,69 @@
+from pycaret.loggers.base_logger import BaseLogger
+from pathlib import Path
+import uuid
+from copy import deepcopy
+import joblib
+
+try:
+    import comet_ml
+except ImportError:
+    comet_ml = None
+
+class CometLogger(BaseLogger):
+    def __init__(self) -> None:
+        if comet_ml is None:
+            raise ImportError(
+                "CometLogger requires Comet. Install using `pip install comet_ml`"
+            )
+        super().__init__()
+        self.run = None
+    
+    def init_experiment(self, exp_name_log, full_name=None, **kwargs):
+        self.run = comet_ml.Experiment(project_name=exp_name_log, **kwargs)
```

Comment:
is there a reason to call it "run" and not "experiment"?

does it have to be public?

Reply:
run is what they used for other loggers and they have made it public. so just am following the maintainer's format